dashboard/controller/users/enable2FA.js

```diff
@@ -14,7 +14,7 @@ module.exports = function enable2FA(req, res) {
 
 	if (req.method == 'POST') {
 
-		req.assert('tokenentry', common.l10n.get('TokenAtLeast', {min_token_size})).len(min_token_size);
+		req.assert('tokenentry', common.l10n.get('TokenAtLeast', min_token_size)).len(min_token_size);
```

Comment:
Have you tested your changes? I think this change will break l10n translation argument

Reply:
Yes, but it is giving the error in lint testing, can I add this file in lint ignore?

Reply:
We cannot add code which is managed by us to lint ignore. There would be no point of adding linter to the project if we do that. However, it should be fine to update the rules of eslint so that it does not detects intentional code as bug.


But in this case, just using this would fix the linter issue:
```
{ min_token_size: min_token_size }
```

Btw, have you verified if your change breaks l10n or not?

Reply:
Yes @NikhilM98, due to this change it is breaking the 2-factor authorization, thanks! for the help!